src/otx/algorithms/classification/adapters/mmcls/utils/builder.py

```diff
@@ -10,7 +10,7 @@
 from mmcv.runner import load_checkpoint
 from mmcv.utils import Config, ConfigDict, get_logger
 
-from otx.algorithms.common.utils.logger import LEVEL
+from otx.utils.logger import LEVEL
 
 logger = get_logger("mmcls")
```

Comment:
Is it okay to use get_logger function like this?

Reply:
Thanks for finding. I missed it. I'll update.

Reply:
fixed at [4051d39](https://github.com/openvinotoolkit/training_extensions/pull/2612/commits/4051d395412953637667133bfea8f918151161c9)